test-utils/store-validator-bin/src/main.rs

```diff
@@ -28,8 +31,20 @@ fn main() {
 
     let store = create_store(&get_store_path(&home_dir));
 
-    let mut store_validator = StoreValidator::default();
-    store_validator.validate(&*store, &near_config.genesis.config);
+    let runtime_adapter: Arc<dyn RuntimeAdapter> = Arc::new(neard::NightshadeRuntime::new(
+        &home_dir,
+        store.clone(),
+        Arc::clone(&near_config.genesis),
+        near_config.client_config.tracked_accounts.clone(),
+        near_config.client_config.tracked_shards.clone(),
+    ));
+
+    let mut store_validator = StoreValidator::new(
+        near_config.genesis.config.clone(),
+        runtime_adapter.get_tries(),
```

Comment:
could do `ShardTries::new(store, num_shards)` unless we need RuntimeAdapter for something else

Reply:
I need RuntimeAdapter to call `num_shards()` then.

Reply:
RuntimeAdapter gets it from genesis:
`genesis.config.num_block_producer_seats_per_shard.len() as NumShards`

Reply:
Do you think that's a good idea to bypass standard and transparent way of getting the data and use some weird unclear way to use length of the array of seats per shard as `num_shards` just to save a couple of lines of code in the separate util?